cmd/tink-worker/cmd/root.go

```diff
@@ -134,68 +121,40 @@ func NewRootCommand(version string, logger log.Logger) *cobra.Command {
 	rootCmd.Flags().StringP("registry-password", "p", "", "Sets the registry-password (REGISTRY_PASSWORD)")
 	must(rootCmd.MarkFlagRequired("registry-password"))
 
+	rootCmd.Flags().String("tinkerbell-grpc-authority", "", "Connection info for tink-server (TINKERBELL_GRPC_AUTHORITY)")
```

Comment:
"Connection info" is a little ambiguous, maybe we can call it the tink server endpoint?

Reply:
done